src/coprocessor/dag/batch_executor/table_scan_executor.rs

```diff
@@ -310,482 +312,296 @@ mod tests {
     use crate::coprocessor::dag::expr::EvalConfig;
     use crate::coprocessor::util::convert_to_prefix_next;
     use crate::storage::{FixtureStore, Key};
-    use cop_datatype::FieldTypeAccessor;
-    use cop_datatype::FieldTypeTp;
+    use cop_datatype::{EvalType, FieldTypeAccessor, FieldTypeTp};
     use kvproto::coprocessor::KeyRange;
     use std::sync::Arc;
     use tipb::expression::FieldType;
     use tipb::schema::ColumnInfo;
 
-    #[test]
-    fn test_basic() {
-        // Table Schema: ID (INT, PK), Foo (INT), Bar (FLOAT, Default 4.5)
-        // Column id:    1,            2,         4
-
-        const TABLE_ID: i64 = 7;
+    /// Table Schema: ID (INT, PK), Foo (INT), Bar (FLOAT, Default 4.5)
+    /// Column id:    1,            2,         4
+    struct TableScanTestWrapper {
```

Comment:
Maybe call it `Helper` is better because it doesn't wrap the test, instead it helps the test.